platform/windows/display_server_windows.cpp

```diff
@@ -3451,6 +3509,8 @@ DisplayServer::WindowID DisplayServerWindows::_create_window(WindowMode p_mode,
 		ImmReleaseContext(wd.hWnd, wd.im_himc);
 
 		wd.im_position = Vector2();
+
+		// XXX FIXME this is wrong in cases where the window coordinates were changed due to full screen mode; use WindowRect
```

Comment:
What does "XXX" mean?

Reply:
Sorry just like NOTE those are my tags XXX, NOTE, REVISIT

They slip out of my hands automatically and should not be in your code.

Reply:
To answer your question XXX in my code bases is "do not ship if build includes this tag", but yeah that was by mistake.

Reply:
removed